### PR TITLE
fix(alert): close button styling

### PR DESCRIPTION
**Pull request by damianOfKyan**
### Proposed behaviour
Requires #2690 to be merged before it is tested.

Screenshot 2020-03-04 at 12 25 09

Update the Alert close icon, when using a modern theme the icon will have a gold outline.
Corrects alignment and adds Cypress feature tests.

Screenshot 2020-03-04 at 12 13 56

### Current behaviour
The icon has a blue outline.

### Checklist

- [x] Release notes (Conventional Commits) 
- [x] Unit tests
- [x] Cypress automation tests
<del>- [x] Storybook added or updated
<del>- [x] Theme support
<del>- [ ] Typescript `d.ts` file added or updated

### Additional context
There are other outstanding PR's related to close icon.
#2690  #2691 #2692 #2693

### Testing instructions
1. `npm start`
1. Go to http://localhost:9001/?path=/story/alert--with-button

### CODE REVIEW

**Comment by damianOfKyan** [comment]
> Hello @damianOfKyan ! :)
> Probably I'm wrong but I can't find the IconButton in the code. Is that correct?

yes, IconButton was merged after this PR was made, I just updated PR by rebasing on master.

File: cypress/features/regression/alert.feature [hunk 1]
```diff
@@ -4,6 +4,12 @@ Feature: Alert component
   Background: Open Alert component page
     Given I open "Alert" component page with button
 
+  @positive
+  Scenario: CloseIcon has the border outline
+    When I open component preview
+    Given closeIcon is focused
+    Then closeIcon has the border outline
```
Comment (DlgSHi) [inline cypress/features/regression/alert.feature]:
```
@positive
   Scenario: CloseIcon has the border outline
     Given I open component preview
     When closeIcon is focused
     Then closeIcon has the border outline

File: cypress/features/regression/alertClassic.feature [hunk 2]
```diff
@@ -4,9 +4,8 @@ Feature: Alert component classic story
...
-  @ignore
-  @FE-2508
+  @positive
   Scenario: CloseIcon has the border outline
     When I open component preview
-      And I hit Tab key 1 time
-    Then closeIcon has no border outline for classic story
\ No newline at end of file
+    Given closeIcon is focused
+    Then closeIcon has border outline for classic story
```
Comment (DlgSHi) [inline cypress/features/regression/alertClassic.feature]: it should be `Given` - `When` - `Then`

Comment (damianOfKyan) [reply cypress/features/regression/alertClassic.feature]: fixed: d1c3ae1

File: cypress/features/regression/alert.feature (hunk 1 above)
Comment (damianOfKyan) [reply cypress/features/regression/alert.feature]: fixed: d1c3ae1

File: src/components/alert/alert.style.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+import StyledIconButton from '../icon-button/icon-button.style';
+
+const AlertStyle = styled.div`
+  ${StyledIconButton} {
+    position: absolute;
+    top: 25px !important;
```
Comment (edleeks87) [inline src/components/alert/alert.style.js]: Can we use `&&` instead of `!important` here?

Comment (damianOfKyan) [reply src/components/alert/alert.style.js]: fixed: 387f75c

**Pull request merged**